driver.md

```diff
@@ -24,8 +24,9 @@ Some Ethereum commands take an Ethereum specification (eg. for an account or tra
     syntax EthereumSimulation ::= ".EthereumSimulation"
                                 | EthereumCommand EthereumSimulation
  // ----------------------------------------------------------------
-    rule .EthereumSimulation => .
-    rule ETC:EthereumCommand ETS:EthereumSimulation => ETC ~> ETS
+    rule <k> .EthereumSimulation                                 => .                   ... </k>
+    rule <k> ETC                          ETS:EthereumSimulation => ETC          ~> ETS ... </k>
+    rule <k> ETC1:EthereumCommand ~> ETC2 ETS:EthereumSimulation => ETC1 ~> ETC2 ~> ETS ... </k>
 
```

Comment:
@dwightguth I'm particularly interested in whether you have a better solution to this problem. I need `__` to simplify out in the second position here as well.